src/shadowbox/scripts/update_mmdb.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Download the GeoLite2 country database into the same location
+# used by Alpine's libmaxminddb package.
+
+TMPDIR="$(mktemp -d)"
+FILENAME="GeoLite2-Country.mmdb"
+LIBDIR="/var/lib/libmaxminddb"
+curl --silent "https://geolite.maxmind.com/download/geoip/database/$FILENAME.gz" -o "$TMPDIR/$FILENAME.gz"
+gunzip "$TMPDIR/$FILENAME.gz"
+mkdir $LIBDIR
```

Comment:
Use `mkdir --parents` so it doesn't fail if the directory already exists.

Reply:
Done.  (Alpine uses busybox, which only support the short form.)